library_generation/test/owlbot/fix_poms_unit_tests.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import shutil
+import unittest
+from library_generation.owlbot.src.fix_poms import main
+from library_generation.test.compare_poms import compare_xml
+from library_generation.test.test_utils import FileComparator
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+resources_dir = os.path.join(script_dir, "..", "resources", "test-owlbot")
+file_comparator = FileComparator()
+
+
+class FixPomsTest(unittest.TestCase):
+    def test_fix_poms_update_poms_for_ad_manager_correctly(self):
```

Comment:
The test case is not specific to `ad_manager`, more like `non-cloud`, or `group_id_does_not_start_with_google` to be more specific.

Reply:
Done.